### PR TITLE
Load all the existing aspects from the capsule before running install

### PR DESCRIPTION
**Pull request by zkochan**
And alternative solution to #5404

## Proposed Changes

-
-
-

### CODE REVIEW

File: scopes/dependencies/pnpm/lynx.ts [hunk 1]
```diff
@@ -226,6 +229,23 @@ export async function install(
   }
 }
 
+async function includeAllComponentsFromDir (rootDir: string, manifestsByPaths) {
+  const files = await fs.readdir(rootDir, { withFileTypes: true })
+  return Promise.all(
+    files
+      .filter((file) => file.isDirectory() && file.name !== 'node_modules')
+      .map((dir) => path.join(rootDir, dir.name))
+      .filter((dirPath) => !manifestsByPaths[dirPath])
```
Comment (GiladShoham) [inline scopes/dependencies/pnpm/lynx.ts:238]: add another filter to check if there is a pacakge.json file.
We create some other folders in the root sometimes which are not capsules, and won't have package.json

**Review by GiladShoham — CHANGES_REQUESTED** [review]

**Comment by GiladShoham** [comment]
I'm not sure this should be in the lynx file. as this is pnpm specific, while this might happen for yarn as well. 
If we read everything, we should do it in a more general place

File: scopes/dependencies/pnpm/lynx.ts (hunk 1 above)
Comment (zkochan) [reply scopes/dependencies/pnpm/lynx.ts:238]: it is covered at line 241.

if the `package.json` is not found, such error is ignored at line 243

**Comment by zkochan** [comment]
I think this should be done for Yarn as well. Because Yarn has the same issue. I can move this code to some utility and call from both Yarn and pnpm

**Review by davidfirst — APPROVED** [review]

**Pull request merged**